boomerang-io/webapp-packages
# Feature add community link

#### Changelog

**New**

- Add `help` navbar menu link to Community url

**Changed**

- Update color theme to match carbon 10 color palette as seen [inVision designs](https://ibm.invisionapp.com/d/main?redirHash=#/console/15354319/319324138/preview)

**Removed**

- Remove duplicated styles

#### Testing / Reviewing

1. View the UIShell branch [deploy preview](https://deploy-preview-11--carbon-addons-boomerang-react.netlify.app/)
3. Test out all of the UIShell features to make sure they are still visual coherent - nav links, menu, sidenav, modals etc
4. See how you feel about all of the hover/active/focus states for all of the elements
5. To test the active header nav link, right click on the link, inspect, in the styles add add class `--is-active`
